common/postgresql-ng/docker/entrypoint.sh

```diff
@@ -51,17 +51,21 @@ if [[ $(id -u) == 0 ]]; then
   done
 
   # do a one time migration from old postgres chart
-  if [[ -e /data/PG_VERSION ]]; then
-    old_version=$(cat /data/PG_VERSION)
+  if [[ -e /data/data/PG_VERSION ]]; then
+    old_version=$(cat /data/data/PG_VERSION)
 
     # move to a temporary directory
     mkdir -p /data-old
-    mv /data/* /data-old/
+    mv /data/data/* /data-old/
+    rmdir /data/data
```

Comment:
Only seeing this now, but this will be very expensive and slow for the large DBs (Arc and Sentry) because you're moving off NFS into local storage and then back into NFS. Would it be possible to have the temp location within the NFS volume?